libqtile/widget/battery.py

```diff
@@ -191,6 +226,17 @@ class _LinuxBattery(_Battery, configurable.Configurable):
             None,
             "Name of file with the current power draw in /sys/class/power_supply/battery_name",
         ),
+        (
+            "charge_controller",
```

Comment:
I think you need to reference this in the docstring i.e. give examples of how to use it (including the `thunderbolt_smart_charge` function which I would keep in this module).

You may also want to be clear that the default behaviour (where this is supported) is to charge to 90% - given that this is a change to current behavior.

Reply:
Sounds good. I updated the doc string, switched the default back to 100% (i.e. `charge_controller is None`), and added a comment in the changelog.